TravelerApp/TravelerAppCore/Controller/SearchByName.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TravelerAppCore.Models.Hotels;
+using TravelerAppCore.Controller;
+using TravelerAppCore.View;
+using System.Text.RegularExpressions;
+
+namespace TravelerAppCore.Controller
+{
+    public class SearchByName
+    {
+      
+
+        public static List<int> HotelsByName(List<Root> dataRead, string searchedName)
+        {
+            List<int> foundHotels = new List<int>();
+            int exactMatch = 0;
+            //check if a search name has at least one character in it
+
```

Comment:
jak chcesz sprawdzić czy string ma jakieś znaki to możesz użyć string.IsNullOrWhiteSpace()